lib/net.c

```diff
@@ -1789,20 +1929,30 @@ drop_privileges(void)
 		return -1;
 	}
 
-	if (initgroups(pw->pw_name, config.pw_gid) != 0) {
-		G_LOG(ERR, "%s: failed to call initgrous(%s, %u) - %s\n",
-			__func__, pw->pw_name, config.pw_gid, strerror(errno));
+	G_LOG(NOTICE, "Ambient capabilities supported: %s\n",
+		CAP_AMBIENT_SUPPORTED() ? "yes" : "no");
+
+	G_LOG(NOTICE, "Gatekeeper pid=%u\n", getpid());
+
+	log_proc_caps("Starts with");
```

Comment:
The log entry `"Gatekeeper pid=%u\n"` should go immediately before the call of `log_ratelimit_enable()` in `finalize_stage2()` since it is useful independently of capabilities.